README.md

```diff
@@ -210,3 +210,13 @@ Checklist:
     - ensure it is added as a content file to the `DemoServer` project
     - for non-C# demos: make sure that the `Slug` value matches the corresponding C# demo slug
     - ensure that `SourceFileName` matches the source code file name located in the same folder as `metadata.json`. Please remember to include the file extension.
+
+## When adding a new C# demo:
+
+1. Add demo to `demos.json`
+2. Add slug to `slugs.ts`
+3. Add the demo `controller.cs` file
```

Comment:
Maybe `controller.cs` should start with upper case? (`Controller.cs`)

Reply:
done